pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -19,10 +19,17 @@
 , gtest
 , protobuf3_21
 , pythonSupport ? true
+, cudaSupport
+, cudaPackages
+, cudaCapabilities ? cudaPackages.cudaFlags.cudaCapabilities
+, symlinkJoin
```

Comment:
Unused?

Reply:
Yes, at some point during the PR there was a symlinked environment  - removed this and other unused attributes.